components/ai_chat/core/browser/utils.cc

```diff
@@ -80,6 +81,18 @@ void SetUserOptedIn(PrefService* prefs, bool opted_in) {
   }
 }
 
+bool IsBraveSearchSERP(const GURL& url) {
+  if (!url.is_valid()) {
+    return false;
+  }
+
+  // https://search.brave.com/search?q=test
+  return url.SchemeIs(url::kHttpsScheme) &&
+         url.host_piece() == "search.brave.com" &&
```

Comment:
Also can use `brave_domains::GetServicesDomain("search")` here too so that we can test this against staging/dev search

Reply:
Addressed in https://github.com/brave/brave-core/pull/25257/commits/b17b5128d6f3b7fd82477e6070095ac2190b8549